modules/mapi-capi-migration-overview.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cluster_api_machine_management/cluster-api-disabling.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="mapi-capi-migration-overview_{context}"]
+= Migrating Cluster API resources to Machine API resources
+
+On clusters that support migrating between Machine API and Cluster API resources, the two-way synchronization controller supports converting a Cluster API resource to a Machine API resource.
+
+[NOTE]
+====
+The two-way synchronization controller only operates on clusters with the `MachineAPIMigration` feature gate in the `TechPreviewNoUpgrade` feature set enabled.
+====
+
+You can migrate resources that you originally migrated from the Machine API to the Cluster API, or resources that you created as Cluster API resources initially.
+Migrating an original Machine API resource to a Cluster API resource and then migrating it back provides an opportunity to verify that the migration process works as expected.
+
+[NOTE]
+====
+You can only migrate some resources on supported infrastructure types.
+====
+
+.Supported resource conversions
+[cols="6",options="header"]
+|===
+|Infrastructure
```

Comment:
What are you referring to with `Infrastructure`? The platform's VMs?

Reply:
Yeah, so historically a lot of the docs say "cloud provider" but my understanding is that's only accurate for e.g. AWS, GCP, etc. Was thinking along the lines of "the instance on the underlying infrastructure" <-- that use of "infrastructure", but I am open to other ideas (I know we overuse that word in here and I want it to be clear :sweat_smile:)